docs/pages/versions/unversioned/sdk/audio.md

```diff
@@ -406,7 +406,7 @@ try {
   - `canRecord` : a boolean set to `true`.
   - `isRecording` : a boolean describing if the `Recording` is currently recording.
   - `durationMillis` : the current duration of the recorded audio.
-  - `metering` : a number that's the most recent reading of the loudness in dB. Present or not based on Recording options. See `RecordingOptions` for more information.
+  - `metering` : a number that's the most recent reading of the loudness in dB. The value ranges from –160 dBFS, indicating minimum power, to 0 dBFS, indicating maximum power. Present or not based on Recording options. See `RecordingOptions` for more information.
```

Comment:
> Present or not based on Recording options

I know you didn't add this phrase, but do you know what it means? I don't think I understand it

Reply:
Basically, `metering` is an indication of volume (loudness) here

Reply:
Yes- `metering` makes sense to me, but the phrase "Present or not based on Recording options" confused me. I see now that it means it is only included in the result if `isMeteringEnabled ` was providing in the RecordingOptions